FrontEnd/src/components/basics/Fallback/Fallback.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FunctionComponent } from 'react';
+import { amenityToIcon, Amenity } from '../Amenities/Amenities';
+import { Icon as IconType, miscIcons } from '@icons';
+
+// Amenity needs to be any so it can be type guarded
+interface FallbackProps {
+  amenity: any;
```

Comment:
If I understand correctly, FallbackIcon is meant to be a Fallback _only_ for the following case: when something that was meant to be an icon (ie received an icon key from the backend) is actually not in the list of icons on the frontend.

So in this case, we should only pass in the type of `Amenity`. This FallbackIcon _expects_ an Amenity, but will provide a _Fallback_ in case the string passed through is actually not an Amenity. Does that make sense? Reach out if not and I can explain further